scipy/stats/vonmises_cython.pyx

```diff
@@ -30,7 +30,7 @@ cdef double von_mises_cdf_series(double k,double x,unsigned int p):
         return 0.5 + x / (2 * NPY_PI) + V / NPY_PI
 
 cdef von_mises_cdf_normalapprox(k, x):
-    b = sqrt(NPY_2_PI) * np.exp(k) / i0(k)
+    b = sqrt(NPY_2_PI) / i0e(k) # Check for negative k
```

Comment:
I think the idea is that the new code is not equivalent to the old code when `k` is negative, because an absolute value is taken.  But for the Von-Mises distribution `k` must be non-negative anyway so it's OK.